tests/sqllogictests/suites/temp_table/temporary_tables_table.sql

```diff
@@ -0,0 +1,22 @@
+statement ok
+CREATE TEMP TABLE t0(a int not null, b int not null);
+
+statement ok
+CREATE OR REPLACE DATABASE d1;
+
+statement ok
+CREATE TEMP TABLE d1.t1(a int not null, b int not null);
+
+statement ok
+CREATE OR REPLACE DATABASE d2;
+
+statement ok
+CREATE TEMP TABLE d2.t2(a int not null, b int not null) ENGINE = Memory;
+
+query T
+select * from system.temporary_tables order by table_id;
+----
+default t0 4611686018427407904 FUSE
+d1 t1 4611686018427407905 FUSE
+d2 t2 4611686018427407906 MEMORY
```

Comment:
maybe we should exclude the `table_id` column? likely to be flaky?

Reply:
The table_id for temporary tables is allocated by incrementing from a fixed starting value, which should be deterministic in this case.

Reply:
got it